test/integ-new/main_new_test.go

```diff
@@ -303,6 +304,38 @@ func TestCertRotatorKeyRotation(t *testing.T) {
 				3: kmspb.CryptoKeyVersion_ENABLED,
 			})
 	})
+
+	t.Run("invalid_primary", func(t *testing.T) {
+		// Set primary to a version that doesn't exist
+		if err := jvscrypto.SetPrimary(ctx, kmsClient, keyResouceName, keyResouceName+"/cryptoKeyVersions/99"); err != nil {
```

Comment:
if we don't set a label or setup a invalid label in ci workflow (https://github.com/abcxyz/jvs/blob/main/.github/workflows/ci-new.yml#L183), the public key's none-breakglass test case will fail. 

Therefore, for these edge cases, I think it's better set label after the normal cases have finished, and then run the edge cases. 

In theory this way the rest of the integ test cases are already finished when cert rotator's test reaches this step. If we have concerns, we can consider sequently running public key service's integ test and then run cert rotator's.